x-pack/plugins/triggers_actions_ui/public/application/sections/test_utils.tsx

```diff
@@ -51,15 +50,13 @@ export const createAppMockRenderer = (
   });
 
   const AppWrapper: React.FC<{ children: React.ReactElement }> = React.memo(({ children }) => (
-    <I18nProvider>
```

Comment:
The i18n context provider is not explicitly needed since KibnaRenderContextProvider provides it